src/StackAdmin/AzureRM/AzureRM.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule             = "AzureRM.psm1"
 
     # Version number of this module.
-    ModuleVersion          = '2.3.0'
+    ModuleVersion          = '2.3.0.1'
```

Comment:
@bganapa nit: any reason this wasn't a patch version bump (`2.3.1`) like `AzureStack.psd1` down below?

Reply:
No particular reason. I just wanted to bump it to something higher.. This is not a version that we will publish anyway. The version bump is just for nightly build consumption..

Reply:
@cormacpayne pls let me know if you want me to change to 2.3.1